test/e2e/utils/enodespecbuilder.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
```

Comment:
We group imports from different sources and add an empty line between two groups.

antrea.io/antrea imports should be put to the last.

```
import (
        crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"

	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
)
```

Reply:
Yes. I'm familiar with it. Missed it in this file. I will update it.